examples/python-dataplugin-examples/csv-read-with-direct-loading/README.md

```diff
@@ -0,0 +1,3 @@
+# CSV With Direct Loading Example
+
+This example shows a DataPlugin that can read the DIAdem example csv files using the standard python `csv.DictReader`. The values are read in all at once and stored directly to the TDM tree.
```

Comment:
> The values are read in all at once

To be more direct, maybe 'The plugin reads all values and stores them...' ?

Reply:
thanks!